docker/Dockerfile

```diff
@@ -196,7 +196,7 @@ ARG MAKEFLAGS="-j2"
 ENV MAKEFLAGS $MAKEFLAGS
 ARG SAGE_NUM_THREADS="2"
 ENV SAGE_NUM_THREADS $SAGE_NUM_THREADS
-RUN make configure
+RUN ./bootstrap
```

Comment:
does `make configure` still work? (if yes maybe leave it alone to be conservative, if not maybe we need to fix it)

Reply:
yes, `make configure` should still work. However, I don't know how, and trying to figure it out gives me a headache. OK, let's revert this change and see.